docs/3_2_model_exchange_api.adoc

```diff
@@ -68,7 +68,7 @@ The model enters *Event Mode* from the *Continuous-Time Mode* and discrete-time
 
 The followings arguments can be set to `fmi3True` to inform the FMU why *Event Mode* was entered.
 
-_[These arguments are not mutually exclusive and may all be `fmi3False` if the caller cannot provide this information.]_
+_[These arguments are not mutually exclusive and may all be `fmi3False`, e.g. for an <<input event>>.]_
```

Comment:
I would not mention fmi3False here at all anymore because the rootsFound argument is not a boolean but an int array.
```suggestion
_[These arguments are not mutually exclusive.]_
```